boot-qemu.sh

```diff
@@ -186,6 +186,15 @@ function setup_qemu_args() {
             QEMU=(qemu-system-ppc64)
             ;;
 
+        riscv)
+            KIMAGE=Image
+            QEMU_ARCH_ARGS=(
+                -bios default
```

Comment:
does this work? I seem to have `-bios /usr/lib/riscv64-linux-gnu/opensbi/qemu/virt/fw_jump.elf` in my shell history.

Reply:
If I use `-bios default` locally, I see:
```
qemu-system-riscv64: Unable to load the RISC-V firmware "opensbi-riscv64-virt-fw_jump.bin"
```

Reply:
It works for me but I build and use QEMU from source. I'll see if I can test with just stock Ubuntu or Debian packages.

Reply:
ah.  I imagine that being an issue for our CI's docker image, which is using ubuntu focal packages.

Reply:
Alright, it should be fixed with 3482733422fc5be0075b802b7eb715bf78b003e2.